api/src/pcapi/core/mails/transactional/users/ubble/ubble_ko_reminder.py

```diff
@@ -19,7 +20,8 @@
 
 
 def send_ubble_ko_reminder_emails() -> None:
-    users = find_users_that_failed_ubble_check_seven_days_ago()
+    days_ago = 7 if settings.IS_PROD else 0
```

Comment:
d'après le message de commit ce serait pas plutôt `if not IS_TESTING` ?

Reply:
si ça ne dépend que de l'env, le plus élégant est peut-être de mettre une variable dans pcapi.settings